### ISSUE TITLE
Use the PyTorch BatchNorm folding instead of a handwritten converter when possible

### ISSUE THREAD

**Issue by narendasan** [NVIDIA/TRTorch#31]
We currently use a handwritten converter for batch norm which fuses the operations into a single convolution. It would be easier to use the `_jit_pass_fold_convbn` pass in lowering instead where we can.

**Comment by narendasan** [comment]
https://github.com/pytorch/pytorch/pull/34835
